pkg/template/resource.go

```diff
@@ -45,14 +45,13 @@ type getTriggerTemplate func(name string, options metav1.GetOptions) (*triggersv
 // ResolveBinding takes in a trigger containing object refs to bindings and
 // templates and resolves them to their underlying values.
 func ResolveBinding(trigger triggersv1.EventListenerTrigger, getTB getTriggerBinding, getTT getTriggerTemplate) (ResolvedBinding, error) {
```

Comment:
should this be called `ResolveBindings` now?

Reply:
Since this resolves Bindings + Template, I don't want it to seem like we'd also support multiple Templates. I think the verbage is fine with respect to handling multiple bindings, but I did go ahead and change `ResolvedBinding.TriggerBinding` -> `ResolvedBinding.TriggerBindings`.